app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -248,6 +248,56 @@ const Thunder = () => {
     </>
   );
 };
+// #endregion
+
+// cloud starts here
+const CloudMotionFrames = keyframes`
+  0% {
+    bottom: 100%
+  }
+  100% {
+    bottom: -12px;
+  }
+`;
```

Comment:
Does the Motion div handle animation speed? What part of the animation does this part control?

Reply:
@tucker-weed it's actually for the movement of the cloud. The above css styling allows vertical movement. However, it needs to move horizontally.